tests/unit/lms/services/lti_registration_test.py

```diff
@@ -0,0 +1,24 @@
+from unittest.mock import sentinel
+
+import pytest
+
+from lms.services.lti_registration import LTIRegistrationService, factory
+from tests import factories
+
+
+class TestLTIRegistrationService:
+    def test_get(self, svc, registration):
+        assert svc.get(registration.issuer, registration.client_id) == registration
+
+    def test_factory(self, pyramid_request):
+        service = factory(sentinel.context, pyramid_request)
+
+        assert isinstance(service, LTIRegistrationService)
+
+    @pytest.fixture
+    def registration(self):
+        return factories.LTIRegistration()
+
+    @pytest.fixture
+    def svc(self, db_session):
+        return LTIRegistrationService(db_session)
```

Comment:
In the other tests I've been doing recently we've kept the tests for the service and factory tests separate, as in one the service is always patched and in the other it usually isn't. In this case it's so small, and does so little it's not as important, but if you wanted to include the suggestion above about making some assertions about the call made to `LTIRegistrationService` for the factory then it might make more sense.

Reply:
Yup, the `isinstance` assertion was a bit lazy. Changed it to the other style :+1: